wire/internal/wire/wire_test.go

```diff
@@ -576,10 +584,45 @@ func (test *testCase) materialize(gopath string) error {
 	return nil
 }
 
-func runGo(bctx *build.Context, args ...string) error {
+// writeGoMod generates go.mod files for the test package and its dependency.
+// The file structure looks like:
+//
+//	gopath/src/
+//
+//		example.com/
+//
+//			go.mod
+//				replaces dependency with local copied one
+//
+//			... (Packages to be built and tested)
+//				any Go files copied recursively
+//
+//		github.com/google/go-cloud/
+//
+//			go.mod
+//
+//			... (Dependency files copied)
+func writeGoMod(gopath string) error {
```

Comment:
For example, what happens here if the user has go 1.10, which doesn't support modules?

Reply:
The go.mod files should be ignored by Go 1.10 and lower.

Reply:
OK.